scipy/optimize/_minimize.py

```diff
@@ -976,9 +978,25 @@ def _add_to_array(x_in, i_fixed, x_fixed):
     return x_out
 
 
+def _validate_bounds(bounds, x0, meth):
```

Comment:
We already have a similar method in `stats._qmc._validate_bounds`. Maybe there is an opportunity to consolidate that in a common place like `_util`? (I hate that name as it's a great excuse to put unrelated random things in. Maybe there is a better place for that).

But I am happy to merge as is and do this in a follow up.

Reply:
Good idea. I'd appreciate it if you'd do this in a follow-up; I'd be happy to review.

Reply:
Will do 👍